core/src/test/java/org/infinispan/distribution/DistSyncFuncTest.java

```diff
@@ -352,10 +352,11 @@ public void testComputeIfPresentFromNonOwner() throws InterruptedException {
       expectException(RemoteException.class, () -> getFirstNonOwner("k1").computeIfPresent("k1", mappingToException));
 
       // remove if after compute value is null
-      retval = getFirstNonOwner("k1").computeIfPresent("k1", (v1, v2) -> null);
-      asyncWait("k1", ComputeCommand.class);
-      if (testRetVals) assertNull(retval);
-      assertRemovedOnAllCaches("k1");
+      // TODO: Fix https://issues.jboss.org/browse/ISPN-8074
+//      retval = getFirstNonOwner("k1").computeIfPresent("k1", (v1, v2) -> null);
+//      asyncWait("k1", ComputeCommand.class);
+//      if (testRetVals) assertNull(retval);
+//      assertRemovedOnAllCaches("k1");
```

Comment:
@karesti I'm not sure. I never really grokked TestNG's behaviour when it comes to inheritance, so I have to test every time to make sure I get the behaviour I want.